docs/content/tutorial/administrative-tasks.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Administrative Tasks"
+date:
+draft: false
+---
+
+## Rotating TLS Certificates
+
+Credentials should be invalidated and replaced (rotated) as often as possible
+to minimize the risk of their misuse. Unlike passwords, every TLS certificate
+has an expiration, so replacing them is inevitable. When you use your own TLS
+certificates with PGO, you are responsible for replacing them appropriately.
+Here's how.
+
+
+PostgreSQL needs to be restarted after its server certificates change.
```

Comment:
:question: Is there anything we need to call out regarding the `customReplicationTLSSecret` in this section? I believe the process for replacing them is more or less the same as described for `customTLSSecret`, but it may be worth saying so explicitly.

Reply:
Nothing needs to reload or restart when the replication certificate changes. The CA needs to continue to match, so maybe something about that?

Reply:
:thinking: Yeah, that I think that would be helpful, just to be safe. And with that, I think this will be good to go. :+1: